Cargo.toml

```diff
@@ -185,6 +185,7 @@ web-sys = { version = "=0.3.46", features = [
 ]}
 js-sys = "0.3.46"
 wasm-bindgen-futures = "0.4.19"
+syn = { version = "=1.0.65", optional = true }
```

Comment:
I don't believe you can have target specific dependencies like that, and it being optional probably doesn't do anything either.

Reply:
Yeah I guess the `optional` here isn't useful since we already know syn is included upstream.

We do use target specific dependencies with fixed versions already though and they seem to work fine